util/grpc.go

```diff
@@ -4,6 +4,10 @@ import (
 	"strings"
 )
 
+const (
+	WorkerAddressHeader = "x_worker_eth_addr"
```

Comment:
Rename to `x-worker-eth-addr`, because of [convention](https://http2.github.io/http2-spec/compression.html#static.table.definition).